src/content/app-platform/namespace-configuration/index.md

```diff
@@ -0,0 +1,78 @@
+---
+linkTitle: App namespace metadata
+title: App's namespace metadata
+description: Documentation on how to labelling, annotating the app CR's namespace
+weight: 20
+menu:
+  main:
+    parent: app-platform
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-batman
+aliases:
+  - /reference/app-configuration/
+user_questions:
+  - How can I label the app CR's namespace?
+  - How can I annotate the app CR's namespace?
+---
+
+# App CR's namespace configuration
+
+At some point, you might want to put the specific labels or annotations to the app's namespace to bring benefits from `linkerd` or `kiam`.
```

Comment:
```suggestion
At some point, you might want to put the specific labels or annotations to the app's namespace to bring benefits from `linkerd` or `kiam`. For example, to include everything in a namespace to the Linkerd service mesh. Or, to tell KIAM you do allow Loki to get logs about pods running in this namespace. 
```

Maybe add example to illustrate? Is this correct?

Reply:
Perfect, thanks.